.github/workflows/pull_request.yml

```
@@ -6,12 +6,14 @@ jobs:
   build-chain:
     strategy:
       matrix:
+        os: [ubuntu-18.04, windows-latest]
         java-version: [8, 11]
       fail-fast: false
-    runs-on: ubuntu-18.04
+    runs-on: ${{ matrix.os }}
     name: Maven Build
     steps:
       - name: Free disk space
```

An additional step should be added here as we do for drools https://github.com/kiegroup/drools/blob/7c64a322182f275c811a658763226858b2efa796/.github/workflows/pull_request.yml#L15
```suggestion
      - name: Support longpaths
        if: ${{ matrix.os == 'windows-latest' }}
        run: git config --system core.longpaths true
      - name: Free disk space
```

---

@Ginxo thx!